src/components/Header/Logo.js

```diff
@@ -19,13 +25,25 @@ const Logo = ({ path, logoColour, blue }) => {
       !path.includes('design') &&
       !path.includes('training') &&
       !blue ? (
-        <img role="link" tab-index="0" height="48" src={logo} alt="yld logo" />
+        <YLDLogoImageWrapper>
+          <img
+            role="link"
+            tab-index="0"
```

Comment:
Do we need this tab-index? (also, react uses `tabIndex`)

Reply:
Oh yeah. Weird.

I didn't change this code though - the `<img>` line is the same as the old line 22. I just wrapped it with YLDLogoImageWrapper.

Not sure why this was there in the first place :woman_shrugging: But you're right. I'll remove it.